Dockerfile

```
@@ -39,7 +39,15 @@ RUN sed -i 's/galette.localhost/galette.${HOSTNAME}/' /etc/apache2/sites-availab
 
 # ENVIRONMENT VARIABLES
 ## Galette version
-ENV GALETTE_VERSION 0.9.6
+ENV GALETTE_VERSION 1.0.0
```

Should not that one be the latest stable?
```suggestion
ENV GALETTE_VERSION 1.0.4
```

---

I'd like to build all versions one by one. I know it's slow, but it will help identify problems when moving slow. :)

---

Oh, OK, no problem :)